cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -630,18 +633,40 @@ func (r *TrafficOpsReq) GetConfigFileList() error {
 		}
 
 		r.configFiles[file.Name] = &ConfigFile{
-			Name: file.Name,
-			Path: filepath.Join(file.Path, file.Name),
-			Dir:  file.Path,
-			Body: []byte(file.Text),
-			Uid:  atsUid,
-			Gid:  atsGid,
-			Perm: mode,
+			Name:     file.Name,
+			Path:     filepath.Join(file.Path, file.Name),
+			Dir:      file.Path,
+			Body:     []byte(file.Text),
+			Uid:      atsUid,
+			Gid:      atsGid,
+			Perm:     mode,
+			Warnings: file.Warnings,
+		}
+		for _, warn := range file.Warnings {
+			if warn == "" {
+				continue
+			}
+			if _, ok := r.configFileWarnings[file.Name]; ok {
```

Comment:
Nitpick: the check isn't necessary, the first argument to `append` is allowed to be nil:
https://go.dev/play/p/fV0pfvWe40z

So this can just always
```
r.configFileWarnings[file.Name] = append(r.configFileWarnings[file.Name], warn)
```
Even if `r.configFileWarnings[file.Name]` is `nil`.

Reply:
Check has been removed.